### PR TITLE
Add moving to open source whitepaper engage page

### LINKED ISSUE #7338: [Whitepaper] Moving to OpenSource - Build takeover and engage page
## Requirements

Please build a takeover and an engage page based on this [brief](https://docs.google.com/spreadsheets/d/1bkM14vEn8noJcQVphj9B_iGV6suh6NYoJncmG-5e_D4/edit#gid=0) and [design](2453):
 
~~build takeover~~
- [x] build engage page
~~add takeover to `/takeovers/index.html`~~
~~add takeover to `/index.html`~~
- [x] add engage page to `/engage/index.html`
- [ ] test the engage page on [https://socialdebug.com/](https://socialdebug.com/) or [http://debug.iframely.com/](http://debug.iframely.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

### PR DESCRIPTION
**Pull request by deadlight**
## Done

- Added a new engage page from this [brief](https://docs.google.com/spreadsheets/d/1bkM14vEn8noJcQVphj9B_iGV6suh6NYoJncmG-5e_D4/edit#gid=1271849439)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/engage/moving-to-opensource-whitepaper
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)

## Issue / Card

Fixes #7338

### CODE REVIEW

**Comment by pmahnke** [comment]
Sorry, @deadlight I had not reviewed the copy.

I think we should change it to:

# Embracing the open source mandate
#### 85% of enterprises have an open source mandate, preference or are exploring

**Also note, no ', or'**

**Comment by deadlight** [comment]
@pmahnke no problem, thanks. I thought that it might get some pushback in QA

**Comment by deadlight** [comment]
@pmahnke ready for re-review

File: templates/engage/index.html [hunk 1]
```diff
@@ -977,14 +977,21 @@ <h1>Ubuntu case studies, whitepapers and webinars</h1>
...
+
     <div class="row u-equal-height u-clearfix">
     {% with title="What&rsquo;s new in Ubuntu 20.04 LTS?",
       description="Learn all about the major features introduced as part of the Ubuntu 20.04 LTS release",
       slug="20.04-webinars",
           type="webinar" %}
       {% include "engage/_article-card.html" %}
     {% endwith %}
+
+    {% with title="Embracing the enterprise open source mandate",
```
Comment (pmahnke) [inline templates/engage/index.html:989]: you didn't update these.

File: templates/engage/index.html [hunk 2]
```diff
@@ -977,14 +977,21 @@ <h1>Ubuntu case studies, whitepapers and webinars</h1>
...
+      description="85% of enterprises report an open source mandate, preference, or exploratory approach ",
```
Comment (pmahnke) [inline templates/engage/index.html:990]: or these

**Comment by deadlight** [comment]
@pmahnke sorry, I forgot that that was hard-coded.

**Pull request merged**